contrib/cirrus/runner.sh

```diff
@@ -43,6 +45,15 @@ _run_setup() {
     "${SKOPEO_PATH}/${SCRIPT_BASE}/runner.sh" setup
 }
 
+_run_debian_setup() {
+    msg "Installing previously downloaded/cached Docker packages"
+    ooe.sh dpkg -i \
+        $PACKAGE_DOWNLOAD_DIR/containerd.io*.deb \
+        $PACKAGE_DOWNLOAD_DIR/docker-ce*.deb
```

Comment:
Is this necessary for c/image / Skopeo? I wouldn’t expect either one to need `docker` if `podman` is installed, and I don’t think anything uses `containerd`.

Reply:
I dunno. I only put it here because the old `.cirrus.Dockerfile` installed some package called `docker.io`.  If you know for sure it's unnecessary, I'll gladly strip it out.

Reply:
AFAICS that originally comes from c000d6b2673e3c8a07dcd93d628ad2f2e0d2e4c3 , when we were running `test-skopeo` inside that container (and Podman was before 1.0).

Anyway, it’s certainly fine for this PR to only migrate the current behavior , and not do other improvements just because they touch the same areas.

Reply:
Thanks for confirming and checking the history.  I'll strip it out because it means entirely avoiding a call to the runner script.